Test/Web/FunctionalTests/TestApps/ConsoleAppFW40/ApplicationInsights.config

```diff
@@ -16,9 +16,7 @@
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.BuildInfoConfigComponentVersionTelemetryInitializer, Microsoft.AI.WindowsServer"/>
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.DeviceTelemetryInitializer, Microsoft.AI.WindowsServer"/>
   </TelemetryInitializers>
-  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel">
-    <EndpointAddress>http://LocalHost:4002/v2/track</EndpointAddress>
-  </TelemetryChannel>
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel"/>
```

Comment:
This will break the Func tests. (Ai.Config is destroyed after every nuget upgrade) Please revert this file to original.